Gemfile

```diff
@@ -35,7 +35,7 @@ gem 'http_accept_language'
 gem 'identity-hostdata', github: '18F/identity-hostdata', tag: 'v4.4.2'
 gem 'identity-logging', github: '18F/identity-logging', tag: 'v0.1.1'
 gem 'identity_validations', github: '18F/identity-validations', tag: 'v0.7.2'
-gem 'jsbundling-rails', '~> 1.1.2'
+gem 'jsbundling-rails', '~> 1.1'
```

Comment:
[question] oh, why was this change necessary?

Reply:
Ah, I meant to acknowledge this. `npm` support was not added in `jsbundling-rails` until [v1.2.2](https://github.com/rails/jsbundling-rails/releases/tag/v1.2.2)

Reply:
thanks, i figured something like this!